TraceScripts/README.md

```diff
@@ -92,4 +92,26 @@ $> adb push <filepath>/PlayFabMultiplayerLogger.json /sdcard/PlayFabLogs/config/
     Please note that if you've modified the `logFolder` property of your `PlayFabMultiplayerLogger.json`, then you'll need to update the ADB command accordingly.
 ```
 $> adb pull /sdcard/PlayFabLogs/log/ <target local directory>
-```
\ No newline at end of file
+```
+
+### Linux: Instructions
+
+1. Download the [`PlayFabMultiplayerLogger.json` file](Linux/PlayFabMultiplayerLogger.json) from this repo.
+2. Open a new terminal.
+3. Create your `config` directory on the Linux device.
+```
+$> mkdir $HOME/PlayFabLogs/config/
+```
+4. You will need to modify the `logFolder` property of the `PlayFabMultiplayerLogger.json` file.
+```
+"logFolder": "/home/<your-username>/PlayFabLogs/log/"
+```
+- Please replace `<your-username>` with your username on your Linux machine. If you are unsure of your username:
+```
+$> echo $HOME
+```
+5. Copy the `PlayFabMultiplayerLogger.json` file into your `$HOME/PlayFabLogs/config/` directory.
+```
+$> cp <filepath>/PlayFabMultiplayerLogger.json $HOME/PlayFabLogs/config/
+```
+6. Run the application. If using the default configuration of `PlayFabMultiplayerLogger.json`, logs should automatically be written to `/home/<your-username>/PlayFabLogs/log/`.
```

Comment:
I think you should mention the 'logFolder' property rather than having an if using default configuration clause.

Reply:
that's what I was thinking as well, but I do already mention it in step 4, so wasn't sure

Reply:
Actually I see that your step 7 was same for Android, so, feel free to bring this back for Linux also.

Reply:
I'm gonna change it to Scott's suggestion above, that's much cleaner imo